hack/test-integration.sh

```diff
@@ -121,10 +127,10 @@ for test in "${tests[@]}"; do
 		fi
 	done
 done
-test_end_time=$(date +%s%3N)
+test_end_time=$(date +%s)
```

Comment:
Isn't this formatting needed for jUnit? Can we confirm it is being generated appropriately now?